### DISCUSSION TITLE
openssl_sign(): Supplied key param cannot be coerced into a private key

### DISCUSSION THREAD

**Discussion by hbynlsl** [yansongda/pay#615] [Q&A]
openssl_sign(): Supplied key param cannot be coerced into a private key ，这是什么原因呢？

**Comment by yansongda** [comment]
检查： app_secret_cert    mch_secret_cert

**Comment by hbynlsl** [comment]
多谢回复。



这样配置，有问题吗？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yansongda/pay"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月25日(星期三) 晚上10:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yansongda/pay] openssl_sign(): Supplied key param cannot be coerced into a private key (Discussion #615)





 
检查： app_secret_cert    mch_secret_cert
